worlds/doom_1993/Items.py

```diff
@@ -0,0 +1,853 @@
+# This file is auto generated. More info: https://github.com/Daivuk/apdoom
+
+from BaseClasses import ItemClassification
+from typing import TypedDict, Dict, Set 
+
+
+class ItemDict(TypedDict, total=False): 
+    classification: ItemClassification 
+    count: int 
+    progression_count: int 
+    name: str 
+    doom_type: int # Unique numerical id used to spawn the item. 
+    episode: int # Relevant if that item targets a specific level, like keycard or map reveal pickup. 
+    map: int 
+
+
+item_table: Dict[int, ItemDict] = {
+    350000: {'classification': ItemClassification.progression,
+             'count': 1,
+             'progression_count': 0,
+             'name': 'Hangar',
+             'doom_type': -1,
+             'episode': 1,
+             'map': 1},
+    350001: {'classification': ItemClassification.filler,
+             'count': 1,
+             'progression_count': 0,
+             'name': 'Hangar - Computer area map',
+             'doom_type': 2026,
+             'episode': 1,
+             'map': 1},
+    350002: {'classification': ItemClassification.progression,
+             'count': 1,
+             'progression_count': 0,
+             'name': 'Nuclear Plant',
+             'doom_type': -1,
+             'episode': 1,
+             'map': 2},
+    350003: {'classification': ItemClassification.progression,
+             'count': 1,
+             'progression_count': -858993460,
```

Comment:
you use
```py           
for i in range(0, count):
  classification = item["classification"]
  if i < item["progression_count"]:
```
so `i` is an incrementing number from 0 to `count` so it makes no sense to have a negative number here. 

same for many other items below